src/chef-mover/src/mv_oc_chef_authz.hrl

```diff
@@ -16,10 +16,10 @@
 -type oc_authz_id() :: <<_:256>>.
 
 -record(authz_group, {actors = [] :: actor_list(),
-		      groups = [] :: group_list()}).
+		              groups = [] :: group_list()}).
```

Comment:
should be alined under the `actors`

Reply:
Weird.  It was aligned perfectly when I brought it up in vim.  Then I discovered some weird invisible characters (not spaces) which were apparently throwing off the alignment here.

Reply:
Fixed.

Reply:
it still seems to be misaligned.